### DISCUSSION TITLE
Installations-Problem

### DISCUSSION THREAD

**Discussion by Wollie99** [Tichau/FileConverter#312] [Q&A]
Hallo!

Folgendes ist keine Frage, sondern eine Feststellung mit einer Bitte an den Produzenten.

Problem:

1.) Installiere ich FileConverter (FC) als lokaler User (also nicht als Admin), muss ich einige Schritte trotzdem als Admin bestätigen.

2.) Ist FC als lokaler User installiert worden, kann ich zwar konfigurieren und dies abspeichern. Doch werden diese Einstellungen im Kontext-Menü gar nicht aufgelistet, denn dort wird ausschließlich die Konfigurations-Option angezeigt (also keinerlei voreingestellte Konvertierungs-Optionen). Der User wird somit automatisch immer wieder zum Konfigurations-Fenster geführt.

3.) Installiere ich FC als Admin, funktioniert die Anwendung.

Also:

Es fehlt der absolut wichtige Hinweis, dass FC nur als Administrator installiert werden kann. Bitte dies auf Ihren Seiten vermerken!

Oder ist mir ein Fehler unterlaufen?

Wollie

-----------------------------------------------------------------------------------------------------------------------------------------

Google-Übersetzung (Google Translate):

The following is not a question but a statement with a request to the producer.

Hello!

Problem:

1.) If I install FileConverter (FC) as a local user (i.e. not as admin), I still have to confirm some steps as admin.

2.) If FC has been installed as a local user, I can configure and save this. However, these settings are not listed at all in the context menu, because only the configuration option is displayed there (ie no preset conversion options). The user is thus automatically led to the configuration window again and again.

3.) If I install FC as admin, the application works.

So:

The absolutely important note that FC can only be installed as an administrator is missing. Please note this on your pages!

Or did I make a mistake?

Wollie

**Comment by Tichau** [comment]
You're right, it can only we installed as an administrator, I will add it the the readme file._
